kong/plugins/prometheus/exporter.lua

```diff
@@ -24,6 +24,10 @@ local function init()
   metrics.db_reachable = prometheus:gauge("datastore_reachable",
                                           "Datastore reachable from Kong, " ..
                                           "0 is unreachable")
+  metrics.upstream_target_health = prometheus.gauge("upstream_target_health",
```

Comment:
first thing I notice here is that there is a `.`, which is supposed to be a `:`, if compared to the line above